internal/controller/progressive/progressive.go

```diff
@@ -406,17 +396,5 @@ func CalculateScaleMinMaxValues(object map[string]any, podsCount int, pathToMin,
 		newMin = newMax
 	}
 
-	// If min was not found, return nil to later restore the appropriate desired value
-	outMin := &min
-	if !foundMin {
-		outMin = nil
-	}
-
-	// If max was not found, return nil to later restore the appropriate desired value
-	outMax := &max
-	if !foundMax {
-		outMax = nil
-	}
-
-	return newMin, newMax, outMin, outMax, nil
+	return newMin, newMax, nil
```

Comment:
I wonder if this function could be worthy of a quick unit test? (to handle the different possibilities, such as `min` not specified)